hadoop-hdds/common/src/test/java/org/apache/hadoop/hdds/utils/db/cache/TestTableCacheImpl.java

```diff
@@ -88,6 +89,55 @@ public void testPartialTableCache() {
   }
 
 
+  @Test
+  public void testPartialTableCacheWithNotContinousEntries() throws Exception {
+    int totalCount = 0;
+    int insertedCount = 3000;
+    for (long i=0; i<insertedCount; i+=2) {
```

Comment:
what is the rationale of this +2 ?

Reply:
Just adding entries in the cache which are not in continuous order.

Reply:
why?

Reply:
And also my initial intention was to build epoch list of some count from this loop, and call cleanup cache. But later added a test with manual entries for epochs.

Reply:
Are you trying to simulate multi-threaded action ? if so, should you be generating a random number and modding by the range? I am not sure +1 or +2 buys you any special testing here.

Reply:
I just want to add entries in cache not in continous order and cleanup to be called with entries not in continous order. And as said my initial intention is to use this for loop to build epoch list for cleanup with some count. I can remove it if you prefer i++?

Reply:
Updated test to use according to my original intention of i+=2